### PR TITLE
CAF-3198 Extract Workflow Initialization

### PR DESCRIPTION
**Pull request by kploch**

### CODE REVIEW

File: utils/data-processing-family-task-submitter/pom.xml [hunk 1]
```diff
@@ -23,7 +23,7 @@
     <parent>
         <groupId>com.github.cafdataprocessing</groupId>
         <artifactId>processing-utilities-aggregator</artifactId>
-        <version>1.1.0-SNAPSHOT</version>
+        <version>1.2.0-SNAPSHOT</version>
     </parent>
```
Comment (mmaBel) [inline utils/data-processing-family-task-submitter/pom.xml:27]: This probably should be in separate commit as it's not related to this work. Not too worried about it though.

File: utils/data-processing-task-submitter/src/main/java/com/github/cafdataprocessing/utilities/tasksubmitter/Main.java [hunk 2]
```diff
@@ -18,13 +18,11 @@
 import com.fasterxml.jackson.databind.ObjectMapper;
```
Comment (mmaBel) [inline utils/data-processing-task-submitter/src/main/java/com/github/cafdataprocessing/utilities/tasksubmitter/Main.java]: Remove the unused imports due to code being moved.

**Review by mmaBel — CHANGES_REQUESTED** [review]
Needs a README in the initialization project folder, similar to the other util projects, would need some instructions for getting started in that file too so consumers know where to start. Aside from that and unused imports it looks good.

File: utils/data-processing-family-task-submitter/pom.xml (hunk 1 above)
Comment (kploch) [reply utils/data-processing-family-task-submitter/pom.xml:27]: Maybe, but I'm not going to move. It was wrong, so it's corrected now :)

**Comment by kploch** [comment]
Added a most basic README. We'll extend it at later point.

File: utils/data-processing-task-submitter/src/main/java/com/github/cafdataprocessing/utilities/tasksubmitter/Main.java (hunk 2 above)
Comment (kploch) [reply utils/data-processing-task-submitter/src/main/java/com/github/cafdataprocessing/utilities/tasksubmitter/Main.java]: Forgot to clean this up :) Done.

**Review by mmaBel — APPROVED** [review]
